### PR TITLE
LU-18 input all updated

### PR DESCRIPTION
**Pull request by archan96**

### CODE REVIEW

File: src/components/Inputs/PinInput/components/pinInputField.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  onKeyDown: (event: React.KeyboardEvent<HTMLInputElement>) => void;
+  fillType: "fill" | "transparent" | "outlined" | "bottom";
+}
+
+const PinInputFieldComponent: ForwardRefRenderFunction<
+  HTMLInputElement,
+  PinInputFieldsProps
+> = ({ index, value, onChange, onKeyDown, fillType = "fill" }, ref) => {
+  const PinInputField = styled.input`
+    width: 40px; /* Adjust width as needed */
```
Comment (sanketXynes) [inline src/components/Inputs/PinInput/components/pinInputField.tsx]: Static values should be in the css file

Comment (sanketXynes) [inline src/components/Inputs/PinInput/components/pinInputField.tsx]: Should use vh, vw instade of px

**Review by sanketXynes — CHANGES_REQUESTED** [review]
Please resolve

File: src/components/Inputs/PinInput/components/pinInputField.tsx (hunk 1 above)
Comment (xynes-admin) [reply src/components/Inputs/PinInput/components/pinInputField.tsx]: Resolved in the new commit.

Comment (xynes-admin) [reply src/components/Inputs/PinInput/components/pinInputField.tsx]: Resolved in the latest commit.

**Review by sanketXynes — APPROVED** [review]
Approved

**Pull request merged**